scripts/circle-ci/run-circle-tests.sh

```diff
@@ -26,7 +28,8 @@ done
 case $CIRCLE_NODE_INDEX in
     0) ./gradlew --profile --continue check $CONTAINER_0_EXCLUDE_ARGS ;;
     1) ./gradlew --profile --continue ${CONTAINER_1[@]} ;;
-    2) ./gradlew --profile --continue ${CONTAINER_2[@]} ;;
+    2) ./gradlew --profile --continue ${CONTAINER_2[@]} -x :atlasdb-ete-tests:longTest ;;
```

Comment:
This is kind of sad, but we discussed and couldn't think of an easy way to make it nicer.